e2e-tests/src/test/java/uk/nhs/adaptors/gp2gp/e2e/EhrExtractTest.java

```diff
@@ -236,16 +285,13 @@ private Document theDocumentTaskUpdatesTheRecord(String conversationId) {
         return getFirstDocumentIfItHasObjectNameOrElseNull(gpcAccessDocument);
     }
 
-    private Document emptyDocumentTaskIsCreated(String conversationId) {
+    private List<Object> emptyDocumentTaskIsCreated(String conversationId) {
```

Comment:
Why Object? are there different items that documents being returned?

Reply:
it's a method that isn't being used, I have removed it.